packages/core/src/utils/nuxt/index.ts

```diff
@@ -26,11 +26,6 @@ export const integrationPlugin = (pluginFn: NuxtPlugin) => (nuxtCtx: NuxtContext
     const injectInContext = createAddIntegrationToCtx({ tag, nuxtCtx, inject });
     const config = getIntegrationConfig(nuxtCtx, configuration);
 
-    const { middlewareUrl, ssrMiddlewareUrl } = nuxtCtx.$config;
```

Comment:
This code was ALWAYS overriding whatever was set in the getIntegrationConfig function because the middlewareUrl is required. If so, then the `newUrl` is completely redundant. Once we get rid of it we can set any url we want to, also `/api` (because Axios will work just fine with it)

Reply:
Yes, it was a dead code. `new URL` couldn't work without middlewareUrl and if there is a middlewareUrl then output of `new URL` was anyway overwritten